string root_link_name
string tip_link_name
---
sensor_msgs/JointState joint_state
